.travis.yml

```diff
@@ -19,6 +19,7 @@ script:
 - rm itest.env
 - mkdir -p ${HOME}/.ssh
 - mv -f id_rsa_dockerfile_image_update ${HOME}/.ssh/id_rsa
+- git config -l
```

Comment:
Is this for debugging?

Reply:
Yep. Removed it in c5db720. Should be good to go @afalko